src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -68,6 +68,8 @@ public string ClientVersion
 
         public IPlatformProxy PlatformProxy { get; internal set; }
 
+        public InternalMemoryTokenCacheOptions TokenCacheAccessorOptions { get; set; }

```

Comment:
@bgavrilMS Rename this to `TokenCacheOptions`?

Reply:
TokenCache is such an overloaded word... How about AccessorOptions?

Reply:
Sorry, should've added my reasoning. 
I thought `TokenCacheAccessorOptions` is too specific (options affect the cache, not just accessing it); `AccessorOptions` is too ambiguous (accessor for what?). Maybe `InternalTokenCacheOptions`? Otherwise, I would leave it at `TokenCacheAccessorOptions`. Either way, it's an internal class.